lib/trento/databases/events/database_instance_registered.ex

```diff
@@ -11,7 +11,6 @@ defmodule Trento.Databases.Events.DatabaseInstanceRegistered do
   defevent version: 2 do
     field :database_id, Ecto.UUID
     field :sid, :string
-    field :tenant, :string
```

Comment:
Just to double check with you.
It is right to remove fields from already existing events, right?

Reply:
Yes, it will just get ignored, no need to upcast